src/providers/python.rs

```diff
@@ -80,6 +91,19 @@ impl Provider for PythonProvider {
 
         Ok(None)
     }
+    fn environment_variables(
+        &self,
+        app: &App,
+        _env: &Environment,
+    ) -> Result<Option<EnvironmentVariables>> {
+        if app.includes_file("poetry.lock") {
+            return Ok(Some(EnvironmentVariables::from([(
+                "POETRY_VERSION".to_string(),
+                "1.1.13".to_string(),
```

Comment:
> Also, why is the version specified as an env var. Is it not specified in the requirements.txt file? Instead of using an env var I think we should just use a version by default and allow overriding the version with an env var.

There's no `requirements.txt` in projects that use poetry as a package manager.